subsys/partition_manager/flash_map_partition_manager.c

```diff
@@ -9,7 +9,7 @@
 #include <pm_config.h>
 #include <sys/util.h>
 
-#define FLASH_MAP_OFFSET(i) UTIL_CAT(PM_, UTIL_CAT(PM_##i##_LABEL, _ADDRESS))
+#define FLASH_MAP_OFFSET(i) (UTIL_CAT(PM_, UTIL_CAT(PM_##i##_LABEL, _ADDRESS)) - PM_ADDRESS)
```

Comment:
@hakonfam I will remove this commit when fix will be ready

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/4991

Reply:
I removed this commit. we have to wait until #4991 will go in